src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralDTO.kt

```diff
@@ -40,7 +40,8 @@ class SentReferralDTO(
         relevantSentenceId = referral.relevantSentenceId!!,
         actionPlanId = referral.currentActionPlan?.id,
         currentActionPlanId = referral.currentActionPlan?.id,
-        approvedActionPlanIds = referral.actionPlans?.filter { it.approvedAt != null }?.map { it.id },
+        approvedActionPlanIds = referral.actionPlans?.filter { it.approvedAt != null }
+          ?.sortedByDescending { it.approvedAt }?.map { it.id },
```

Comment:
Do we need these IDs? Could they be replaced by calling `/sent-referral/{id}/approved-action-plans` (which should probably have the same DTO as `/action-plan/{id}`)

Reply:
(And the `.../approved-action-plans` could have a query filter for optimisation)

Reply:
yeh you can get that info from that existing endpoint. as you say, it's probably preferable that way. 

i made this tweak locally and wondered what you thought. honestly, the main reason i did so is because you have to write painful logic to order the dates on the javascript side, vs calling `sortedBy....` in kotlin.

Reply:
also, for what it's worth, i would probably expect most recent first for a field like this (if i was expecting some kind of order)

Reply:
👍

to avoid ambiguity, can we remove this field? given they're just IDs, the UI would have to dereference its content anyway

if we need to keep it, can we document the order for swagger?

Reply:
👍